private/bufpkg/bufmodule/bufmoduleref/bufmoduleref.go

```diff
@@ -69,6 +69,13 @@ type FileInfo interface {
 	isFileInfo()
 }
 
+// InconsistentDigestError is an interface usable with errors.Is to determine
+// if module pin digests have changed unexpectedly.
+type InconsistentDigestError interface {
+	Error() string
+	inconsistentDigest()
```

Comment:
Unexported field with an exported interface. Why restrict who can instantiate this type of error?

Reply:
Just doesn't make sense to raise this elsewhere. I was trying to do the equivalent of a `var ErrInconsistentDigest` but allow a custom error message.

Reply:
Changed this implementation to be more sane.